waspc/cli/src/Wasp/Cli/Command/CreateNewProject/Common.hs

```diff
@@ -13,11 +12,5 @@ import qualified Wasp.Version as WV
 throwProjectCreationError :: String -> Command a
 throwProjectCreationError = throwError . CommandError "Project creation failed"
 
-throwInvalidTemplateNameUsedError :: Command a
-throwInvalidTemplateNameUsedError =
-  throwProjectCreationError $
-    "Are you sure that the template exists?"
-      <> " 🤔 Check the list of templates here: https://github.com/wasp-lang/starters"
```

Comment:
Moved this from common because:
- It's very specific and only used in a single place.
- The function was missing information to output a meaningful error message (chosen template name, available templates). Both things are available in the caller.